interpreter/memory_metering_test.go

```diff
@@ -109,12 +115,16 @@ func TestInterpretArrayMetering(t *testing.T) {
 		assert.Equal(t, uint64(0), meter.getMemory(common.MemoryKindAtreeArrayMetaDataSlab))
 		assert.Equal(t, uint64(8), meter.getMemory(common.MemoryKindAtreeArrayElementOverhead))
 		assert.Equal(t, uint64(1), meter.getMemory(common.MemoryKindElaboration))
-		assert.Equal(t, uint64(4), meter.getMemory(common.MemoryKindVariable))
-		// 1 Int8 for type
-		// 2 String: 1 for type, 1 for value
-		// 3 Bool: 1 for type, 2 for value
-		assert.Equal(t, uint64(6), meter.getMemory(common.MemoryKindPrimitiveStaticType))
-		assert.Equal(t, uint64(10), meter.getMemory(common.MemoryKindVariableSizedStaticType))
+
+		// TODO: assert equivalent for compiler/VM
+		if !*compile {
+			assert.Equal(t, uint64(4), meter.getMemory(common.MemoryKindVariable))
+			// 1 Int8 for type
+			// 2 String: 1 for type, 1 for value
+			// 3 Bool: 1 for type, 2 for value
+			assert.Equal(t, uint64(6), meter.getMemory(common.MemoryKindPrimitiveStaticType))
+			assert.Equal(t, uint64(10), meter.getMemory(common.MemoryKindVariableSizedStaticType))
```

Comment:
Can we already assert these two (i.e: static types) for the compiler/VM. I guess only the `MemoryKindVariable` is not available?

Reply:
`MemoryKindVariableSizedStaticType` is also zero, not quite sure why